hail-ci-build.sh

```diff
@@ -4,12 +4,10 @@ set -ex
 PROJECTS='hail batch ci site scorecard cloudtools'
 
 for project in $PROJECTS; do
-    CHANGED=$(python3 project-changed.py target/$TARGET_BRANCH $project)
-    if [[ $CHANGED != no ]]; then
-	if [[ -e $project/hail-ci-build.sh ]]; then
+    if [[ -e $project/hail-ci-build.sh ]]; then
+        CHANGED=$(python3 project-changed.py target/$TARGET_BRANCH $project)
+        if [[ $CHANGED != no ]]; then
```

Comment:
I see you're using `!= no` rather than `== yes`. do we want to be aware of CHANGED taking a value other than yes or no?

Reply:
The logic is: if something goes wrong I'd rather redeploy than not.

Reply:
In particular, not deploying is just an optimization to save work and/or disruption.